controllers/front/CartController.php

```diff
@@ -98,6 +98,10 @@ public function initContent()
             Tools::redirect('index.php');
         }
 
+        // check that minimal quantity conditions are respected for each product in the cart
+        if (!Tools::getValue('ajax')) {
```

Comment:
Any reason to don't check it in ajax context?

Reply:
Ajax calls on the cart page also go through this method, but does its own checks later and sends back a raw html template to update the page.

Here we want to do this check when the page is loading (or being refreshed), just in case someone comes back to a cart he created earlier, and minimal quantities required for a product changed since then.

Reply:
Then you should add a comment to explain this behaviour

Reply:
good idea, just did it 👍